pkg/storage/fs/owncloud/owncloud.go

```diff
@@ -175,7 +179,7 @@ func (c *config) init(m map[string]interface{}) {
 		c.Redis = ":6379"
 	}
 	if c.UserLayout == "" {
-		c.UserLayout = "{{.Username}}"
+		c.UserLayout = "{{.Id.OpaqueId}}"
```

Comment:
This changes the default, but will prevent username changes from breaking the storage lookup.

Reply:
I put this there to align to the change in ocis-reva which also changes the default. I hope it's ok.
Otherwise we'll have further discrepancies.